controllers/apply/reaching-communities/processor.js

```diff
@@ -21,17 +21,34 @@ module.exports = async function processor({ form, data, stepsWithValues, mailTra
         template: path.resolve(__dirname, './customer-email.njk'),
         templateData: {
             data: data,
+            stepsCopy: copy.steps,
+            fieldsCopy: copy.fields,
             summary: stepsWithValues,
             isArray: xs => Array.isArray(xs)
         }
     });
 
+    /**
+     * This is slightly awkward but allows us to reorder the steps for internal use
+     * We need to reorder both the steps and the copy to make sure the labels are in sync
+     */
+    const stepsCopy = copy.steps;
+    const stepsCopyInternalOrder = [stepsCopy[3], stepsCopy[2], stepsCopy[1], stepsCopy[0]];
+    const stepsWithValuesInternalOrder = [
+        stepsWithValues[3],
+        stepsWithValues[2],
+        stepsWithValues[1],
+        stepsWithValues[0]
+    ];
```

Comment:
I think this is okay – hopefully in the future we'll be less dependent on emails for storing data so we won't need this kind of functionality.